### ISSUE TITLE
Altitude

### ISSUE THREAD

**Issue by lindakuhnz** [mbari-media-management/vars-annotation#72]
ROV altitude is collected, but is not populated in VARS. This information is biologically relevant. Can we add these data to VARS? There are (were) fields available in the gui and query.

**Comment by lindakuhnz** [comment]
Hold on- what we are logging for altitude may not be what I thought it was (ROV height off the seafloor). I'm checking with Rich.

**Comment by hohonuuli** [comment]
For most analysis I've done in the past, I haven't trusted the ROV altitude. I use a bathymetric grid and calculate altitude using the ROV lat, long, depth and the depth of the grid at that location. Currently, no altitude is being stored in M3_ANNOTATIONS although there is a column for it.

**Comment by lindakuhnz** [comment]
Yes - we (ROV pilots) have been working on this. It is available, and is accurate these days. Working with Rich to get it logged somewhere. Many users of VARS don't have access to or know how to use grids, ArcGIS, etc.

**Comment by hohonuuli** [comment]
OK, just let me know when Rich adds it to EXPD. Once, that's done I'll add it to the merge.
